spec/plugins/url_plugin_spec.rb

```diff
@@ -12,24 +12,27 @@
 #testing with no url
 no_url = {}
 
+#TODO: test for the retry limit feature & connect timeout
 describe Ragios::Plugin::UrlMonitor do
   it "should send a http GET request to the url and pass" do
     regular_url_plugin = Ragios::Plugin::UrlMonitor.new
     regular_url_plugin.init(regular_url)
     regular_url_plugin.test_command?.should == true
+    regular_url_plugin.test_result.should == {"HTTP GET Request to #{regular_url[:url]}" => 200}
   end
 
   it "should send a http GET request to the url and fail" do
     failing_plugin = Ragios::Plugin::UrlMonitor.new
     failing_plugin.init(failing_url)
     failing_plugin.test_command?.should ==  false
+    failing_plugin.test_result.should == {"HTTP GET Request to http://www.google.com/fail/"=>"Expected([200, 201]) <=> Actual(404 Not Found)\n"}
```

Comment:
Line is too long. [144/80]<br>Surrounding space missing for operator `=>`.<br>Space inside { missing.<br>Space inside } missing.